src/core/mac/sub_mac.cpp

```diff
@@ -105,15 +105,15 @@ otRadioCaps SubMac::GetCaps(void) const
     caps |= OT_RADIO_CAPS_ENERGY_SCAN;
 #endif
 
-#if OPENTHREAD_CONFIG_MAC_SOFTWARE_TX_SECURITY_ENABLE
+#if OPENTHREAD_CONFIG_MAC_SOFTWARE_TX_SECURITY_ENABLE && (OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2)
```

Comment:
Should also change the default value to 1 in `examples/platforms/simulation/openthread-core-simulation-config.h` ?

https://github.com/openthread/openthread/blob/4484042ced953b9d24a47cc8f3a04ea0e531e18c/examples/platforms/simulation/openthread-core-simulation-config.h#L118

Reply:
I think You are wright. Fixed this.